docs/spec.md

```diff
@@ -446,18 +435,19 @@ to that of the derivation (or derivations).
 
 
 
-## Traversable
-
-#### Dependencies
-
-  1. Functor
-  1. Foldable
+### Traversable
 
-#### Methods
+In the following signature `Applicative<U>` means that a value must
+not only match `Applicative` signature, but also fully support `Applicative` algebra.
```

Comment:
How about this? Probably not optimal, but it's a little more specific, and ties to my suggestion below.

> In the following signature, `U<*> extends Applicative<U>` means that values of that type must be modules that both implement the `Applicative` signature and follow its laws. `U` itself represents the type of such values.